LadybugTools_Adapter/AdapterActions/Execute.cs

```diff
@@ -83,11 +83,15 @@ private List<object> RunCommand(GetMaterialCommand command)
             {
                 JsonFile = new FileSettings()
                 {
-                    FileName = $"LBTBHoM_Materials_{DateTime.Now:yyyyMMdd}.json",
+                    FileName = $"LBTBHoM_Materials.json",
                     Directory = Path.GetTempPath()
                 }
             };
 
+            TimeSpan timeSinceLastUpdate = DateTime.Now - File.GetCreationTime(config.JsonFile.GetFullFileName());
+            if (timeSinceLastUpdate.Days > 30)
```

Comment:
We should probably make this configurable for people?

Reply:
Could I do this through an ActionConfig? I feel like this could be a general setting for the adapter as well.